Gemfile

```diff
@@ -6,4 +6,5 @@ group :test do
   # TODO: change test not to use shoulda.
   #
   gem "shoulda"
+  gem "rubocop"
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)